pskb_website/views.py

```diff
@@ -628,12 +635,18 @@ def save():
                                             author_real_name=user.name)
 
     if not article:
-        flash('Failed creating guide on github', category='error')
-        return redirect(url_for('index'))
+        redirect_to = url_for('index')
+        data = {'error': 'Failed creating guide on github', 'redirect': redirect_to}
+        return Response(response=json.dumps(data), status=500, mimetype='application/json')
 
+    # TODO: move this to another endpoint
```

Comment:
Good point.  I've added a new [issue](https://github.com/pluralsight/guides-cms/issues/33) to refactor this out of the main save endpoint.